src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -320,10 +349,10 @@ struct SvgParser
 
 struct SvgLoaderData
 {
-    vector<SvgNode *> stack;
+    SvgVector<SvgNode *> stack = {nullptr, 0, 0};
```

Comment:
Is there any reason to initialize only on this line?
Don't initialize another SvgVector<T>?

Reply:
others are get calloc() subsequent initialization